use-cases/school-timetabling/src/main/java/org/acme/schooltimetabling/solver/TimetableConstraintProvider.java

```diff
@@ -9,11 +9,7 @@
 import ai.timefold.solver.core.api.score.stream.Joiners;
 
 import org.acme.schooltimetabling.domain.Lesson;
-import org.acme.schooltimetabling.solver.justifications.RoomConflictJustification;
-import org.acme.schooltimetabling.solver.justifications.StudentGroupConflictJustification;
-import org.acme.schooltimetabling.solver.justifications.StudentGroupSubjectVarietyJustification;
-import org.acme.schooltimetabling.solver.justifications.TeacherConflictJustification;
-import org.acme.schooltimetabling.solver.justifications.TeacherRoomStabilityJustification;
+import org.acme.schooltimetabling.solver.justifications.*;
```

Comment:
Please configure your IDE to not use * imports; that is a Solver convention. Enumerating imports is helpful for the JVM, as it doesn't need to load entire packages, and it is also helpful for the reviewer, to see what exactly is being used.